### PR TITLE
First draft of museum

### PR DESCRIPTION
**Pull request by guthrie-kuckes**

### CODE REVIEW

File: 01-python/leisure/museum.py [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+    print("\n\n")
+    for artist in artists:
+        printFrame();
+        print(artist)
+        print()
+    print("\n\n")
+
     return
+
+
```
Comment (gallettilance) [inline 01-python/leisure/museum.py]: Nit: a bit too many new lines here

File: 01-python/leisure/museum.py [hunk 2]
```diff
@@ -1,3 +1,32 @@
...
+artists=["Rodin", "Van Gogh", "Pollack", "Miró "]
+
 def draw_museum():
-    print("museum not found")
+    print("Welcome to the Boston University Computer Science Museum")
+    
+    print("\n\n")
+    for artist in artists:
+        printFrame();
```
Comment (gallettilance) [inline 01-python/leisure/museum.py]: Why not add `height` and `width` as parameters to `draw_museum` and pass them to `printFrame`?

File: 01-python/leisure/museum.py [hunk 3]
```diff
@@ -1,3 +1,32 @@
+
+
+artists=["Rodin", "Van Gogh", "Pollack", "Miró "]
```
Comment (gallettilance) [inline 01-python/leisure/museum.py]: Why not have this scoped to your `draw_museum` function? Is there a reason you want it available outside that function?

**Comment by guthrie-kuckes** [comment]
I responded to the review and pushed again. Can you see my changes/is this the right way to respond?

File: 01-python/leisure/museum.py [hunk 4]
```diff
@@ -1,3 +1,31 @@
-def draw_museum():
-    print("museum not found")
+
+
+
```
Comment (gallettilance) [inline 01-python/leisure/museum.py]: Please remove these extra lines

File: 01-python/leisure/museum.py [hunk 5]
```diff
@@ -1,3 +1,31 @@
...
+def draw_museum(frameWith=10, frameHeight=10):
+    artists=["Rodin", "Van Gogh", "Pollack", "Miró "]
```
Comment (gallettilance) [inline 01-python/leisure/museum.py]: better to capitalize constants
```suggestion
    ARTISTS=["Rodin", "Van Gogh", "Pollack", "Miró "]
```

File: 01-python/leisure/museum.py [hunk 6]
```diff
@@ -1,3 +1,31 @@
...
+    for artist in artists:
+        printFrame(frameWith,frameHeight);
+        print(artist)
+        print()
+    print("\n\n")
+
+    return
+
+
+def printFrame(width=10, height=10):
```
Comment (gallettilance) [inline 01-python/leisure/museum.py]: what if width or height is 0?

**Comment by gallettilance** [comment]
@guthrie-kuckes do you need extra time to address feedback or should I close this PR?

**Comment by guthrie-kuckes** [comment]
Hi! Sorry for the late reply, please close it (though I think you did).

Best,
Guthrie Kuckes

**Pull request closed without merging**